### DISCUSSION TITLE
Note - Brother plugin installation is broken in 0.8.1 - but seems fixed in the dev. version.

### DISCUSSION THREAD

**Discussion by miggland** [inventree/InvenTree#3557] [General]
I would report a bug, but it seems to only concern 0.8.0/0.8.1.

Maybe it concerns other plugins as well, but have not tested.

I don't want to upgrade my production server to a dev version. Any timing on 0.8.2 yet (I see there isn't much left.. )?

**Comment by matmair** [comment]
We release once development is done. There are no release dates until a release is made.
